src/solver/edges/IngredientEdge.tsx

```diff
@@ -73,6 +73,9 @@ export const IngredientEdge: FC<EdgeProps<Edge<IIngredientEdgeData>>> = ({
     : getBezierPath(edgePathParams);
 
   const duration = 60 / (data?.value ?? 0);
+  const neededBelts = maxBelt
+    ? Math.ceil((data?.value ?? 0) / maxBelt.conveyor!.speed)
```

Comment:
Maybe I'm thinking too much on this one, but I'd like to have the system consider to use the "best" belt, so:
- If value > maxBelt speed, it's ok like this
- but if value is < maxBelt speed, it should try to get the lowest belt with a sufficient speed, e.g. "5/min" -> 1 Mk1
Just to be more helpful

Furthermore we can just say `const maxAvailableBelt = maxBelt ?? last(FactoryConveyorBelts);` so that we have always a valid belt to display (e.g. Mk6 if nothing specified)

Reply:
I'd put this in a dedicated file & function, like a `getIngredientEdgeBelt.ts`

Reply:
Honestly, if you ask me, that's an edge case.  If i plan a factory I just want to know how many belts. II personally work with manifolds, rather than balancers.

Reply:
I work with manifolds too (pioneer efficency ye!),  but I know, it's more a commodity than an issue (a mk5 belt will do if a mk1 belt will do). We can keep it like this for now